backend/services/domainOfActivityService.js

```diff
@@ -0,0 +1,13 @@
+const domainOfActivityDAO = require('../DAO/domainOfActivityDAO');
+
+module.exports = {
+
+    getAllDomainsOfActivity: async () => {
+        return await domainOfActivityDAO.getAllDomainsOfActivity();
+    },
+
+    existsDomainOfActivity: async (id) => {
+        return await domainOfActivityDAO.existsDomainOfActivity(id);
```

Comment:
if you only return what the DAO gets, then you don't need the "async" and "await". You need them only when you are doing something else before the return.

Reply:
same as above